client/components/board/simulatorFactory.js

```diff
@@ -0,0 +1,32 @@
+angular.module('OpenEXP')
+    .factory('simulatorFactory', () => {
+        const OpenBCIBoard = require('openbci-sdk');
+        const ourBoard = new OpenBCIBoard.OpenBCIBoard();
+
+        var storage = {};
+
+        var simulatorStart = () => {
+            ourBoard.simulatorStart()
+                .then(() => {
+                    ourBoard.on('sample',sample => {
+                        storage[sample.sampleNumber] = sample;
```

Comment:
does this mean that all samples will be saved in memory?  we may have to deal with this down the line..